src/app/api/nutrition/goals/route.ts

```
@@ -0,0 +1,121 @@
+      { name: 'daily_fat_goal_g', value: daily_fat_goal_g },
+      { name: 'daily_fiber_goal_g', value: daily_fiber_goal_g },
+    ];
+
+    for (const field of numericFields) {
+      if (field.value === undefined || field.value === null) {
+        return NextResponse.json(
+          { error: `Missing required field: ${field.name}` }, 
+          { status: 400 }
+        );
+      }
+      if (typeof field.value !== 'number' || field.value < 0 || !isFinite(field.value)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)